addons/web/static/tests/views/fields/text_field_tests.js

```diff
@@ -58,6 +57,19 @@ QUnit.module("Fields", (hooks) => {
                         },
                     ],
                 },
+                partner_list: {
+                    fields: {
+                        partner_ids: {
+                            string: "Partners",
+                            type: 'one2many',
+                            relation: 'partner',
+                            relation_field: 'id'
+                        },
+                    },
+                    records: [
+                        {id: 1, partner_ids: [1]},
```

Comment:
please run the lint/prettier tooling before committing: `./addons/web/tooling/enable.sh`